crates/iota/src/client_commands.rs

```diff
@@ -591,43 +591,43 @@ pub struct OptsWithGas {
 
 impl Opts {
     /// Uses the passed gas_budget for the gas budget variable and sets all
-    /// other flags to false, and emit to an empty vector(defaulting to all emit
-    /// options).
+    /// other flags to false, and display to an empty vector(defaulting to all
+    /// display options).
```

Comment:
maybe change the doc comment, because it's not vector anymore.

also ... can we put labels in backticks, so it reads a bit easier?